_data/simple-icons.json

```
@@ -15275,6 +15275,11 @@
             "hex": "CC2936",
             "source": "https://www.zotero.org/support/brand"
         },
+        {
+            "title": "Zsh",
+            "hex": "F15A24",
+            "source": "https://github.com/Zsh-art/logo/blob/7b54f2965cb2fc4d138db332e28283f0c6296850/svg/black_vertical_icon.svg"
```

```suggestion
            "source": "https://github.com/Zsh-art/logo/blob/253927e62bd2a640d29915ad7a794880b787eda8/svg/black_vertical_icon.svg"
```
This seems to be like the latest commit

---

Oh... at the risk of delaying this further (sorry!), should this be using `https://github.com/Zsh-art/logo/blob/main/svg/black_simple_icon.svg` as the source, instead? (Latest commit permalink, `https://github.com/Zsh-art/logo/blob/253927e62bd2a640d29915ad7a794880b787eda8/svg/black_simple_icon.svg`.) That version was added to the repo specifically for compatibility with Simple Icons.

---

@ferdnyc
I think the difference has been made when you tried to compound all paths into one.
As my commit shows, the file that I retried to make and an original SVG file are completely the same.

---

@5ouma 

That's cool, whatever you prefer. If the newer version is the Most Right™ one, feel free to open a PR updating `black_simple_icon.svg` and I'm sure @jdorfman will merge it.